### PR TITLE
Fixed gameserver

### PR DESCRIPTION
**Pull request by jc14**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (calebjanderson) [inline .gitignore:1]: ?

Comment (jc14) [inline .gitignore:1]: idk why that's back...

File: gameServer.js [hunk 2]
```diff
@@ -2,14 +2,15 @@ var express = require('express');
...
-
-app.use(express.static('/Users/calebanderson/Desktop/OregonFail'));
+app.use(express.static(__dirname));
 
 app.get('/', function(err, res) {
 	if (err) {
 		console.log('error')
 	}
-	res.sendFile('/Users/calebanderson/Desktop/OregonFail/index.html')
+	console.log('Root', __dirname);
```
Comment (calebjanderson) [inline gameServer.js]: nice! I was struggling with this 👍

**Comment by calebjanderson** [comment]
Looks good. Just do a bit of clean up (removing console.logs etc) and resubmit

**Comment by jc14** [comment]
Ok console logs are gone

**Pull request merged**